src/Raven.Server/ServerWide/Commands/ClusterTransactionCommand.cs

```diff
@@ -669,11 +672,12 @@ public static IEnumerable<SingleClusterDatabaseCommand> ReadCommandsBatch<TTrans
                         continue;
                     }
 
-                    if (take <= 0)
```

Comment:
why it was moved? it changes paging behavior, now paging was done after all filtering, now it is done before